skr-webhook/templates/deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     metadata:
       labels:
         app: skr-webhook
-        operator.kyma-project.io/pod-restart-trigger: "{{.Values.triggerLabel}}"
+        operator.kyma-project.io/pod-restart-trigger: {{uuidv4}}
```

Comment:
`pod-restart-trigger ` this label we do not need anymore, as we discussed the parsed Manifest will be cached. Could you please remove the trigger label logic?

Reply:
I think we still need it. when the set flags' values change, only the `webhook` config and secret will change and the pods will not be restarted. unless this label value changes.